libraries/AP_Devo_Telem/AP_Devo_Telem.h

```diff
@@ -0,0 +1,80 @@
+// -*- tab-width: 4; Mode: C++; c-basic-offset: 4; indent-tabs-mode: nil -*-
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef __AP_DEVO_TELEM_H__
+#define __AP_DEVO_TELEM_H__
+
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Param/AP_Param.h>
+#include <AP_Math/AP_Math.h>
+#include <AP_GPS/AP_GPS.h>
+#include <AP_AHRS/AP_AHRS.h>
+#include <AP_Baro/AP_Baro.h>
+#include <AP_BattMonitor/AP_BattMonitor.h>
+#include <AP_SerialManager/AP_SerialManager.h>
+
+
+#define DEVOM_SYNC_BYTE        0xAA
+
+#pragma pack(push, 1)
```

Comment:
Instead of this, please use PACKED as that's what we use in the rest of the code base, example: 
https://github.com/ArduPilot/ardupilot/blob/ee8adf90730002b4211bd36dd289d60d8fe4c0a5/libraries/AP_GPS/AP_GPS_UBLOX.h#L126